browser/brave_shields/brave_timestamp_rounding_browsertest.cc

```diff
@@ -0,0 +1,111 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <math.h>
+#include <string>
+
+// Prevent a -Wimplicit-const-int-float-conversion warning:
+#define fmod(A, B) fmod(A, static_cast<double>(kMaxULL) + 1.0)
+#include "third_party/blink/renderer/bindings/core/v8/v8_binding_for_core.h"
+#undef fmod
+
+#include "base/test/scoped_feature_list.h"
+#include "base/test/task_environment.h"
+#include "base/time/time.h"
+#include "brave/components/brave_shields/common/features.h"
+#include "content/public/renderer/render_frame.h"
+#include "content/public/test/render_view_test.h"
+#include "third_party/blink/public/common/features.h"
+#include "third_party/blink/renderer/modules/video_rvfc/video_frame_callback_requester_impl.h"
+
+namespace blink {
+
+namespace {
+
+using features::kBraveRoundTimeStamps;
+
+class BraveTimeStampRoundingRenderViewTest : public content::RenderViewTest {
+ public:
+  double ExecuteJSAndReturnDouble(const std::u16string& script) {
+    double result;
+    EXPECT_TRUE(ExecuteJavaScriptAndReturnNumberValue(script, &result));
+    return result;
+  }
+
+  void CheckRounded(const std::u16string& script, bool expect_rounded) {
+    double result = ExecuteJSAndReturnDouble(script);
+    if (expect_rounded) {
+      EXPECT_EQ(round(result) - result, 0);
```

Comment:
I think we should use `EXPECT_DOUBLE_EQ` for all `double` expects:
https://github.com/google/googletest/blob/main/docs/reference/assertions.md#floating-point-comparison-floating-point

Reply:
Fixed